### PR TITLE
add starwarsblocks script, enableShowCode, use blocklyMessage

### PR DESCRIPTION
**Pull request by cpirich**
- allow playlab levels to turn on the "show code" that is typically disabled for blockly playlab levels. Levels can add `level.enableShowCode` to do this. All blockly hoc2015 levels now have this on. The code is a little odd because it doesn't make it clear which events the code belongs to, but it is still of some interest.
- enhance `level.progressConditions` system to allow a `blocklyMessage` to be specified in addition to `message`. If blockly is being used, it will attempt to use `blocklyMessage` instead of `message`. Rewrote error messages in hoc2015 that referenced JavaScript commands and created alternate `blocklyMessage` versions for the blockly hoc2015 script.
- Created the official script for blockly hoc2015: `starwarsblocks` - made it behave just like the `starwars`script. It currently includes the exact videos from the `starwars` script. This will need to be updated once we have blockly specific videos ready.

### CODE REVIEW

**Comment by cpirich** [comment]
@breville @islemaster

File: dashboard/config/locales/scripts.en.yml [hunk 1]
```diff
@@ -1437,3 +1437,8 @@ en:
           description_short: Learn to program droids, and create your own Star Wars game in a galaxy far, far away.
```
Comment (breville) [inline dashboard/config/locales/scripts.en.yml:1437]: if you don't want to commit this file, staging should generate the same content.  but this should be fine.

File: dashboard/config/scripts/starwarsblocks.script [hunk 2]
```diff
@@ -0,0 +1,32 @@
+title 'Star Wars: Building a Galaxy with Code'
```
Comment (breville) [inline dashboard/config/scripts/starwarsblocks.script:1]: awesome.  worth testing as well?  there are some tests covered in https://github.com/code-dot-org/code-dot-org/commit/32dda69212ec987cfc2f099fefbd89c410618a72 though with fixtures not so much is being tested any more.

**Comment by breville** [comment]
lgtm

**Pull request merged**